lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -47,6 +49,7 @@ def tick_update_position(tick, tickxs, tickys, labelpos):
 
     tick.tick1On, tick.tick2On = True, False
     tick.tick1line.set_linestyle('-')
+    tick.tick1line.set_linewidth(rcParams['grid.linewidth'])
```

Comment:
You are right -- xtick.major.width would be better.  But what about `ztick` -- we don't have rcParams for that.  Should we make them?